tests/test_quantities.py

```diff
@@ -104,12 +105,18 @@ def test_gross_margin(technologies, capacity, market):
     market.prices[:] = prices = 3
     market.prices[{"commodity": is_pollutant(usage)}] = env_prices = 6
     # We expect a xr.DataArray with 1 replacement technology
-    actual = gross_margin(technologies, capacity, market.prices)
+    actual = gross_margin(technologies, capa, market.prices)
 
     revenues = prices * prod * sum(is_enduse(usage))
     env_costs = env_prices * envs * sum(is_pollutant(usage))
     cons_costs = prices * fuels * sum(is_fuel(usage))
-    var_costs = vp * (capa**ve) * market.represent_hours / sum(market.represent_hours)
+    var_costs = (
```

Comment:
As above, it will be clearer if some context is added to the docstrings.

Reply:
Thanks. I added a link and an explanation of the terms.
Also, it is more common to express the gross margin as a percentage of the revenues. Therefore I added this additional calculation.